code/datums/components/acme_wall.dm

```diff
@@ -0,0 +1,27 @@
+/datum/component/acme_wall
+	var/overlay
+
+/datum/component/acme_wall/Initialize(mutable_appearance/overlay)
+	if(!istype(parent, /turf/closed/wall))
```

Comment:
This should be for any movable, there's no wall specific behavior we are relying on